### PR TITLE
PD-874: Toggle Bug

### PR DESCRIPTION
**Pull request by brookescarlett**
## ✍️ Proposed changes

🎟 _Jira ticket:_ [PD-874](https://jira.mongodb.org/browse/PD-874)

## 🛠 Types of changes

- [ ] Tooling (updates to workspace config or internal tooling)
- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## ✅ Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] I have run `yarn changeset` and documented my changes

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README

### CODE REVIEW

File: packages/toggle/src/Toggle.story.tsx [hunk 1]
```diff
@@ -15,6 +15,7 @@ storiesOf('Toggle', module).add('Default', () => {
         padding: 20px;
       `}
     >
+      <div>Brooke</div>
```
Comment (DesignerDave) [inline packages/toggle/src/Toggle.story.tsx]: This probably wasn't an intentional addition.

File: packages/toggle/src/Toggle.tsx [hunk 2]
```diff
@@ -23,13 +23,13 @@ const toggleInput = createDataProp('toggle-input');
...
-const focusRing = 3;
+export const focusRing = 3;
 
 const inputStyle = css`
   margin: 0;
   position: absolute;
-  left: 100%;
-  top: 100%;
+  left: 0;
+  top: 0;
```
Comment (DesignerDave) [inline packages/toggle/src/Toggle.tsx:32]: Just in case, let's add 
```css
height: 0;
width: 0;
margin: 0;
```

**Review by DesignerDave — CHANGES_REQUESTED** [review]

File: packages/toggle/src/Toggle.story.tsx (hunk 1 above)
Comment (brookescarlett) [reply packages/toggle/src/Toggle.story.tsx]: Yes already rm'd

File: packages/toggle/src/Toggle.tsx (hunk 2 above)
Comment (brookescarlett) [reply packages/toggle/src/Toggle.tsx:32]: we have margin but will add height and width

**Review by DesignerDave — APPROVED** [review]

**Pull request merged**